addons/voice/org.openhab.voice.googletts/src/main/java/org/openhab/voice/googletts/internal/GoogleCloudAPI.java

```diff
@@ -229,14 +230,37 @@ File synthesizeSpeech(String text, GoogleTTSVoice voice, String codec) {
         }
     }
 
+    /**
+     * Create cache entry.
+     *
+     * @param text      Converted text.
+     * @param cacheFile Cache entry file.
+     * @param audio     Byte array of the audio.
+     * @throws IOException in case of file handling exceptions
```

Comment:
The formatter in Eclipse will put only one space between the parameter names and description in JavaDocs. So can you adjust your IDE settings?

So whenever some saves this file in Eclipse this is automatically formatted to:

```java
    /**
     * Create cache entry.
     *
     * @param text Converted text.
     * @param cacheFile Cache entry file.
     * @param audio Byte array of the audio.
     * @throws IOException in case of file handling exceptions
     */
```

Reply:
Sure. I'll reformat the entire binding...